src/main/java/seedu/address/ui/CommandInfoDisplay.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.ui;
+
+import static java.util.Objects.requireNonNull;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.TextArea;
+import javafx.scene.layout.Region;
+
+
+public class CommandInfoDisplay extends UiPart<Region> {
```

Comment:
I would suggest prepending this class with the name "Help" so we know it's part of the Help portion of the GUI